examples/accessible.js

```diff
@@ -13,9 +12,9 @@ var map = new ol.Map({
   ],
   target: 'map',
   controls: ol.control.defaults({
-    attributionOptions: /** @type {olx.control.AttributionOptions} */ ({
+    attributionOptions /** @type {olx.control.AttributionOptions} */: {
```

Comment:
This will break things for the Compiler.  I'll open a Prettier issue about this.